apps/sqlparser/sqlparser.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import shutil
+import csv
+import sys
+
+from datetime import datetime
+from pathlib import Path
+from collections import Counter
+
+# constants, can be overriden using arguments
+
+## implied folder structure: modules/*/sql/dbname/*/*.sql
+glob = "**/sql/**/*.sql"
+
+repo_folder = Path("/Users/v0/Py/_hobby/wow/server")
+
+
+default_argparsed_args = ["--help"]
+# default_argparsed_args = "-y", ""  # , "--glob", "test"
+
+source_suffix = "modules"
+source_dir = lambda: repo_folder / source_suffix
+target_suffix = "data/sql/custom"
+target_dir = lambda: repo_folder / target_suffix
+report_suffix = "modules_sql_parsed.csv"
+report_path = lambda: repo_folder / report_suffix
+
+modules_to_ignore = ["mod-playerbots"]
+
+counters = dict(
+    written=0, skipped=0, total=0, modules=Counter()  # module: counter
+)
+
+
+def get_db_name(arg):
+    if arg.startswith("db"):
+        # assumes db_*.sql or db-*.sql filename
+        arg = arg[3:]
+    return f"db_{arg}"
+
+
+def collect_sql_files(full_fp, files_to_ignore=None, dry=False):
+    # fp means filepath
+    global counters
+
+    fp = full_fp.relative_to(source_dir())
+
+    module = fp.parents[-2]
+    path_to_parse = fp.parts[fp.parts.index("sql") + 1 :]
+    database = get_db_name(path_to_parse[0])
+    other_folders = "".join(path_to_parse[1:-1])
```

Comment:
A little bit unclear what this block is doing. 

Short comment above it with the input and expected output would be appreciated.